NitroxServer/GameLogic/EscapePodManager.cs

```diff
@@ -62,10 +67,11 @@ public void RepairEscapePodRadio(NitroxId id)
         private EscapePodModel CreateNewEscapePod()
         {
             int totalEscapePods = EscapePods.Count;
+            RandomStartGenerator randomStart = NitroxServiceLocator.LocateService<RandomStartGenerator>();
```

Comment:
Let's pass this in through the constructor.  You may want to (optionally) seed it on creation instead of passing seed in via the method below.